geoserver/geoserver_seeder/geoserver_seeder.rb

```diff
@@ -8,344 +8,36 @@
 require 'open-uri'
 require 'logger'
 
+require_relative 'layer_seeder'
+require_relative 'geonetwork_connector'
+require_relative 'geowebcache'
+
 $logger = Logger.new(STDOUT)
 $logger.level = Logger::INFO
 
-class TileGenerator
```

Comment:
If this code is merged and gets automatically deployed by the nightly chef-run then won't it break the existing seeder? 

We may want to wait until geowebcache is really ready to go. 

Perhaps it would be worthwhile doing this work as a separate seeder script, so as not to interfere with the existing one - until we're ready to switch.

It's too bad there's no proper testing environment.

Also which node are we intending to run this on? The idea of beanstalking everything would be to remote 1-aws-syd which is where it currently runs.